connectors/services/job_cleanup.py

```diff
@@ -27,6 +28,7 @@ def __init__(self, config):
     async def _run(self):
         logger.debug("Successfully started Job cleanup task...")
         self.connector_index = ConnectorIndex(self.es_config)
+        self.es_management_client = ESManagementClient(self.es_config)
```

Comment:
We use different variable (`es_client`) names in other places. It's a nit request but I would make it the same for all the classes.

Reply:
So use `es_client` for `ESManagementClient` too, or have `es_management_client` variable?

Reply:
I don't have a strong opinion on that :). I think `es_management_client`  is a little more explicit so I would use it but it's up to you. I'm fine with `es_client` too.

Reply:
Makes sense! I've addressed this in https://github.com/elastic/connectors/pull/1999/commits/bf1ef636637bbe307d2e81d8b0b1237db02b7a4c